v1/plugin/flags.go

```diff
@@ -0,0 +1,56 @@
+package plugin
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/urfave/cli"
+)
+
+var (
+	flConfig = cli.StringFlag{
+		Name:  "config",
+		Usage: "config file to use",
+	}
+	// If no port was provided we let the OS select a port for us.
+	// This is safe as address is returned in the Response and keep
+	// alive prevents unattended plugins.
+	flPort = cli.StringFlag{
+		Name:  "port",
+		Usage: "port to listen on",
+	}
+	flLogLevel = cli.IntFlag{
+		Name:  "log-level",
+		Usage: "log level - 0:panic 1:fatal 2:error 3:warn 4:info 5:debug",
+		Value: 2,
+	}
+	flPprof = cli.BoolFlag{
+		Name:  "pprof",
+		Usage: "set pprof",
+	}
+	flTLS = cli.BoolFlag{
+		Name:  "tls",
+		Usage: "enable TLS",
+	}
+	flCertPath = cli.StringFlag{
+		Name:  "cert-path",
+		Usage: "necessary to provide when TLS enabled",
+	}
+	flKeyPath = cli.StringFlag{
+		Name:  "key-path",
+		Usage: "necessary to provide when TLS enabled",
+	}
+	flRootCertPaths = cli.StringFlag{
+		Name:  "root-cert-paths",
+		Usage: fmt.Sprintf("root paths separated by OS-specific list separator (e.g. %v)", filepath.ListSeparator),
```

Comment:
Consider change this line 45 to the following
```
Usage: fmt.Sprintf("root paths separated by OS-specific list separator (e.g. %v)", filepath.ListSeparator),
```
to show separator in human readable form

Before:
   --root-cert-paths value   root paths separated by OS-specific list separator (e.g. 58)
After:
   --root-cert-paths value   root paths separated by OS-specific list separator (e.g. :)

Based on the code, it seems that the character ":" is a separator exactly for my OS, so why "e.g." is used there - it rather suggest that it might be ":", but as a user, I need to ensure by my own.